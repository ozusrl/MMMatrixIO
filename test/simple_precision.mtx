%%MatrixMarket matrix coordinate real general
5 5 13
1 1 3.9062500000005e+05
1 2 9.4391444187502e+04
1 5 -9.4391440804947e+04
2 1 9.3132257461548e-10
2 3 1.8878290442236e+05
3 4 -9.4391444187501e+04
4 1 10.7
4 2 10.8
4 3 10.9
4 4 11.1
4 5 11.2
5 1 11.3
5 5 11.4
