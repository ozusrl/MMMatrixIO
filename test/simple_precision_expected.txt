############### MM  ##############
5 5 13
0 0 390625
0 1 94391.4
0 4 -94391.4
1 0 9.31323e-10
1 2 188783
2 3 -94391.4
3 0 10.7
3 1 10.8
3 2 10.9
3 3 11.1
3 4 11.2
4 0 11.3
4 4 11.4
############### COO ##############
int rowIndices[13] = {
0,
0,
0,
1,
1,
2,
3,
3,
3,
3,
3,
4,
4,
};
int colIndices[13] = {
0,
1,
4,
0,
2,
3,
0,
1,
2,
3,
4,
0,
4,
};
double values[13] = {
390625,
94391.4,
-94391.4,
9.31323e-10,
188783,
-94391.4,
10.7,
10.8,
10.9,
11.1,
11.2,
11.3,
11.4,
};
############### CSR ##############
int rowPtr[6] = {
0,
3,
5,
6,
11,
13,
};
int colIndices[13] = {
0,
1,
4,
0,
2,
3,
0,
1,
2,
3,
4,
0,
4,
};
double values[13] = {
390625,
94391.4,
-94391.4,
9.31323e-10,
188783,
-94391.4,
10.7,
10.8,
10.9,
11.1,
11.2,
11.3,
11.4,
};
############### CSC ##############
int rowIndices[13] = {
0,
1,
3,
4,
0,
13,
1,
3,
2,
3,
0,
3,
4,
};
int colPtr[6] = {
0,
4,
6,
8,
10,
13,
};
double values[13] = {
390625,
9.31323e-10,
10.7,
11.3,
94391.4,
10.8,
188783,
10.9,
-94391.4,
11.1,
-94391.4,
11.2,
11.4,
};
